cmd/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	api "github.com/ChainSafe/gossamer/internal"
+	"os"
+
+	cfg "github.com/ChainSafe/gossamer/config"
+	"github.com/ChainSafe/gossamer/p2p"
+	"github.com/ChainSafe/gossamer/polkadb"
+	"github.com/ChainSafe/gossamer/rpc"
+	"github.com/ChainSafe/gossamer/rpc/json2"
+	log "github.com/inconshreveable/log15"
+	"github.com/naoina/toml"
+)
+
+func main() {
+	f, err := os.Open("./config.toml")
+	if err != nil {
+		panic(err)
+	}
+	defer func() {
+		err = f.Close()
+		if err != nil {
+			log.Error("err closing conn", "err", err.Error())
+		}
+	}()
+
+	srvlog := log.New(log.Ctx{"blockchain": "gossamer"})
+	var config cfg.Config
+	if err = toml.NewDecoder(f).Decode(&config); err != nil {
+		srvlog.Warn("toml error::: %s", err.Error())
+	}
+	srv, err := p2p.NewService(config.ServiceConfig)
+	if err != nil {
+		srvlog.Warn("error starting p2p %s", err.Error())
+	}
+	srvlog.Info("🕸️ starting gossamer blockchain...", log.Ctx{"datadir": config.BadgerDB.Datadir})
```

Comment:
Maybe you had something else in mind for this log, but I feel it should happen before the p2p starts. Please omit `blockchain` <3